Assets/Scripts/PlayerScripts/PlayerMovement.cs

```diff
@@ -396,6 +401,31 @@ private void ResetJump()
         _canSaveJump = true;
     }
 
+    public void SetCanSaveJumpFalse()
+    {
+        _canSaveJump = false;
+    }
+
+    public void SetMovePlayerEnabledTrue()
```

Comment:
Minor comment: This script is pretty big. Can we change these methods to accept a boolean parameter and we set the attribute to that parameter? That way we can condense two methods into one :)